modular_skyrat/master_files/code/modules/mob/living/examine_tgui.dm

```diff
@@ -57,17 +57,27 @@
 	var/headshot = ""
 
 	//  Handle OOC notes first
-	if(preferences && preferences.read_preference(/datum/preference/toggle/master_erp_preferences))
-		var/e_prefs = preferences.read_preference(/datum/preference/choiced/erp_status)
-		var/e_prefs_hypno = preferences.read_preference(/datum/preference/choiced/erp_status_hypno)
-		var/e_prefs_v = preferences.read_preference(/datum/preference/choiced/erp_status_v)
-		var/e_prefs_nc = preferences.read_preference(/datum/preference/choiced/erp_status_nc)
-		var/e_prefs_mechanical = preferences.read_preference(/datum/preference/choiced/erp_status_mechanics)
-		ooc_notes += "ERP: [e_prefs]\n"
-		ooc_notes += "Hypnosis: [e_prefs_hypno]\n"
-		ooc_notes += "Vore: [e_prefs_v]\n"
-		ooc_notes += "Non-Con: [e_prefs_nc]\n"
-		ooc_notes += "ERP Mechanics: [e_prefs_mechanical]\n"
+	if(preferences)
+		if(preferences.read_preference(/datum/preference/toggle/master_erp_preferences))
+			var/e_prefs = preferences.read_preference(/datum/preference/choiced/erp_status)
+			var/e_prefs_hypno = preferences.read_preference(/datum/preference/choiced/erp_status_hypno)
+			var/e_prefs_nc = preferences.read_preference(/datum/preference/choiced/erp_status_nc)
+			var/e_prefs_v = preferences.read_preference(/datum/preference/choiced/erp_status_v)
+			var/e_prefs_mechanical = preferences.read_preference(/datum/preference/choiced/erp_status_mechanics)
+			ooc_notes += "ERP: [e_prefs]\n"
+			ooc_notes += "Non-Con: [e_prefs_nc]\n"
+			ooc_notes += "Hypnosis: [e_prefs_hypno]\n"
+			ooc_notes += "Vore: [e_prefs_v]\n"
+			ooc_notes += "ERP Mechanics: [e_prefs_mechanical]\n"
+			ooc_notes += "\n"
```

Comment:
I don't see why it is necessary to add more indents here. The logic is the exact same. Please revert this.